Release/tests/functional/pplx/pplx_test/pplxtask_tests.cpp

```diff
@@ -1889,7 +1865,187 @@ SUITE(pplxtask_tests)
         }
     }
 
-#endif
+#endif // defined(_WIN32) && (_MSC_VER >= 1700) && (_MSC_VER < 1800)
+
+    TEST(MoveTaskFromAnother)
+    {
+        auto val = pplx::create_task([]() { return 17; });
+        auto val2 = std::move(val);
+        VERIFY_ARE_EQUAL(val2.get(), 17);
+    }
+
+    TEST(CreateTaskMoveFromAnother)
+    {
+        auto val = pplx::create_task([]() { return 17; });
+        auto val2 = create_task(std::move(val));
+        VERIFY_ARE_EQUAL(val2.get(), 17);
+    }
+
+    TEST(InvalidateMoveFromTask)
+    {
+        auto val = pplx::create_task([]() { return 17; });
+        auto val2 = std::move(val);
+        VERIFY_IS_NULL(val._GetImpl());
+    }
+
+    TEST(InvalidateTaskCtorFromMove)
+    {
+        auto val = pplx::create_task([]() { return 17; });
+        auto val2 = task<int>(std::move(val));
+        VERIFY_IS_NULL(val._GetImpl());
+    }
+
+    struct CopyOnlyFunctor
+    {
+        explicit CopyOnlyFunctor(int output) : output(output) {}
+        
+        // disable implicitly-declared move constructor and move assignment operator 
+        // generation by declare defaulted copy operations
+        CopyOnlyFunctor(const CopyOnlyFunctor&) = default;
+        CopyOnlyFunctor& operator=(const CopyOnlyFunctor&) = default;
+        
+        int output = 0;
+        int operator()() const { return output; }
+    };
+
+    TEST(create_task_with_copy_only_functor)
+    {
+        const pplx::task<int> t1(CopyOnlyFunctor(17));
+        const auto t2 = pplx::create_task(CopyOnlyFunctor(22));
+        VERIFY_ARE_EQUAL(t1.get(), 17);
+        VERIFY_ARE_EQUAL(t2.get(), 22);
+    }
+
+#if !(defined(_MSC_VER) && (_MSC_VER >= 1800) && !CPPREST_FORCE_PPLX)
```

Comment:
But when `_MSC_VER` is defined, it can be less than 1800, and this should make this `#if` work.

P.S. This all is just opposite condition from the beginning of the *pplxtasks.h* file, where choose between PPL and pplx is made.